tests/queries/0_stateless/01194_http_query_id.sh

```diff
@@ -4,14 +4,14 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-url="http://${CLICKHOUSE_HOST}:${CLICKHOUSE_PORT_HTTP}/?session_id=test_01194"
 rnd=$RANDOM
+url="${CLICKHOUSE_URL}&session_id=test_01194_$RANDOM"
 
 ${CLICKHOUSE_CURL} -sS "$url&query=SELECT+'test_01194',$rnd,1" > /dev/null
 ${CLICKHOUSE_CURL} -sS "$url&query=SELECT+'test_01194',$rnd,2" > /dev/null
 ${CLICKHOUSE_CURL} -sS "$url" --data "SELECT 'test_01194',$rnd,3" > /dev/null
 ${CLICKHOUSE_CURL} -sS "$url" --data "SELECT 'test_01194',$rnd,4" > /dev/null
 
-${CLICKHOUSE_CURL} -sS "$url" --data "SYSTEM FLUSH LOGS"
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}" --data "SYSTEM FLUSH LOGS"
 
-${CLICKHOUSE_CURL} -sS "$url&query=SELECT+count(DISTINCT+query_id)+FROM+system.query_log+WHERE+current_database+LIKE+currentDatabase()+AND+query+LIKE+'SELECT+''test_01194'',$rnd%25'"
+${CLICKHOUSE_CURL} -sS "${CLICKHOUSE_URL}" --data "SELECT count(DISTINCT query_id) FROM system.query_log WHERE current_database = currentDatabase() AND event_date >= yesterday() AND query LIKE 'SELECT ''test_01194'',$rnd%'"
```

Comment:
What if this query starts, and the query start event has a chance to be recorded in the query_log before this query takes the snapshot of the query log? Then the LIKE condition will also match this query by itself.

You can add `AND query NOT LIKE '%this_exact_query%'`

To allow the test to run in parallel, it might also need to be filtered by the session_id.

Reply:
`query LIKE 'SELECT ''test_01194'',$rnd%'"` won't be matched (no starting `%`)